### ISSUE TITLE
Welcome View - buttons should have button role set

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#160409]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG4.1.1; #Parsing

### Environment and OS details:
Visual Studio Code
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:
1)Open Visual studio code editor
2)Using "Alt+H" key navigate to the "Help" menu bar option .
3)Using "Down" arrow key navigate to the "Welcome" item.
4) Make sure you have the following setting in settings.json: "workbench.startupEditor": "gettingStarted"
5)Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started" and observe that the Get started with VS Code button has a link inside it or not. 

### Actual Result:
Get started with VS Code button has a link inside it.

### Expected Result:
There should not be any link inside the Get started with VS Code button.

![A11y_VisualStudioCodeClient_Welcome Screen_Parsing](https://user-images.githubusercontent.com/98530069/189138582-2848dfdf-c11a-4d62-a330-45cda3d40e8a.png)

**Comment by isidorn** [comment]
@lramos15 can we make sure that we set the role=button on the x in the welcome view.
I believe it is very low hanging and is affecting our grade thus assigning to November. Thanks!
